blocks/src/custom-status/block.js

```diff
@@ -24,6 +24,15 @@ let sideEffectL10nManipulation = status => {
   }
 }
 
+// Set the status to the default custom status.
+subscribe(function () {
+  var status = select('core/editor').getEditedPostAttribute('status');
+  if ( typeof status !== 'undefined' && status !== 'publish' ) { 
+    sideEffectL10nManipulation( getStatusLabel( status ) );
+  }
+})
+dispatch('core/editor').editPost( { status: ef_default_custom_status });
```

Comment:
This line is breaking the status revert described here: https://github.com/Automattic/Edit-Flow/pull/521#issuecomment-548836488

Reply:
Fixed.